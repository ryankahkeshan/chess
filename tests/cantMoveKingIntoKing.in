setup
+ K c1 + k c3
+ p h7
done game human human

move c1 b2

move c1 c2

move c1 d2
resign
