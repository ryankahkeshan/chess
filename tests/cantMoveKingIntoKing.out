8  _ _ _ _
7 _ _ _ _ 
6  _ _ _ _
5 _ _ _ _ 
4  _ _ _ _
3 _ _ _ _ 
2  _ _ _ _
1 _ K _ _ 

  abcdefgh

8  _ _ _ _
7 _ _ _ _ 
6  _ _ _ _
5 _ _ _ _ 
4  _ _ _ _
3 _ k _ _ 
2  _ _ _ _
1 _ K _ _ 

  abcdefgh

8  _ _ _ _
7 _ _ _ _p
6  _ _ _ _
5 _ _ _ _ 
4  _ _ _ _
3 _ k _ _ 
2  _ _ _ _
1 _ K _ _ 

  abcdefgh

8  _ _ _ _
7 _ _ _ _p
6  _ _ _ _
5 _ _ _ _ 
4  _ _ _ _
3 _ k _ _ 
2  _ _ _ _
1 _ K _ _ 

  abcdefgh

Invalid move
Invalid move
Invalid move
Black wins!
Final Score:
White: 0
Black: 1
